src/components/SQFormDialog/SQFormDialogInner.tsx

```diff
@@ -55,7 +56,7 @@ interface SQFormDialogInnerProps {
   /** Whether the tertiary button is disabled (Default: false) */
   isTertiaryDisabled?: boolean;
   /** Callback function invoked when the user clicks the tertiary button */
-  onTertiaryClick?: React.MouseEventHandler<HTMLButtonElement>;
+  onTertiaryClick?: (formikContext: FormikContextType<unknown>) => void;
```

Comment:
Instead of `unknown` I think this would be `Values`?

Reply:
Would this mean that we would have to use a generic much like the SQFormDialog file?

Reply:
> Would this mean that we would have to use a generic much like the SQFormDialog file?

Yep!